scripts/lua/modules/pools/pools_rest_utils.lua

```diff
@@ -485,6 +485,7 @@ function pools_rest_utils.get_pool_members(pools)
 
    for member, details in pairs(cur_pool["member_details"]) do
       details["member"] = member
+      details["member_cidr"] = member:match("([^@]+)")
```

Comment:
Please create a utility function to do this (if not already available) to avoid custom regexp around